### ISSUE TITLE
Many scenarios fail due to `Requested target does not receive transfers`

### ISSUE THREAD

**Issue by agatsoh** [raiden-network/light-client#2524]
## Steps to Reproduce
1. 
2. 

## Expected Result
- 

## Actual Result
PFS1,3,6,9
PFS2,4,5,8
MS4
fail due to 

**replacement fee too low**

**replacement transaction underpriced**

Seems like we are trying to send the transactions to the blockchain too fast in the case the nonce remains the same because the earlier transaction has not propagated and new transactions gas has to be 10% higher than the older ones.

source:

https://github.com/ethers-io/ethers.js/issues/444
https://ethereum.stackexchange.com/questions/67548/ethereum-transaction-fails-in-simultaneous-calls

```
2021-01-22T01:01:30.482Z [info] retryWhile: retrying {
  count: 0,
  interval: 5000,
  error: Error: replacement fee too low (error={"reason":"processing response error","code":"SERVER_ERROR","body":"{\"jsonrpc\":\"2.0\",\"id\":605,\"error\":{\"code\":-32000,\"message\":\"replacement transaction underpriced\"}}\n","error":{"code":-32000},"requestBody":"{\"method\":\"eth_sendRawTransaction\",\"params\":[\"0xf8ca8085012a05f20083019092943ea2a1fed7fdef300da19e97092ce8fdf8bf66a380b8640a798f24000000000000000000000000946da73853fac1441998fd5c14c7029988ffe9b80000000000000000000000000404ebf40d76a32b0dccdf13f3ff8a9a7e0fad1900000000000000000000000000000000000000000000000000000000000001f42da0178b57fae97031c6ba540b293f3847589d1890ab0b62b56b4ca4a87786800e7ba03d9e1fcf9f21fa759b832c588bee92d5b99fbff5ada6a72c71f5336c9a22c961\"],\"id\":605,\"jsonrpc\":\"2.0\"}","requestMethod":"POST","url":"http://geth.goerli.ethnodes.brainbot.com:8545"}, method="sendTransaction", transaction="0xf8ca8085012a05f20083019092943ea2a1fed7fdef300da19e97092ce8fdf8bf66a380b8640a798f24000000000000000000000000946da73853fac1441998fd5c14c7029988ffe9b80000000000000000000000000404ebf40d76a32b0dccdf13f3ff8a9a7e0fad1900000000000000000000000000000000000000000000000000000000000001f42da0178b57fae97031c6ba540b293f3847589d1890ab0b62b56b4ca4a87786800e7ba03d9e1fcf9f21fa759b832c588bee92d5b99fbff5ada6a72c71f5336c9a22c961", code=REPLACEMENT_UNDERPRICED, version=providers/5.0.19)
      at Logger.makeError (/lc/index.js:7523:21)
      at Logger.throwError (/lc/index.js:7532:20)
      at checkError (/lc/index.js:12155:16)
      at JsonRpcProvider.<anonymous> (/lc/index.js:12576:47)
      at step (/lc/index.js:12102:23)
      at Object.throw (/lc/index.js:12083:53)
      at rejected (/lc/index.js:12075:65)
      at runMicrotasks (<anonymous>)
      at processTicksAndRejections (internal/process/task_queues.js:93:5) {
    reason: 'replacement fee too low',
    code: 'REPLACEMENT_UNDERPRICED',
    error: Error: processing response error (body="{\"jsonrpc\":\"2.0\",\"id\":605,\"error\":{\"code\":-32000,\"message\":\"replacement transaction underpriced\"}}\n", error={"code":-32000}, requestBody="{\"method\":\"eth_sendRawTransaction\",\"params\":[\"0xf8ca8085012a05f20083019092943ea2a1fed7fdef300da19e97092ce8fdf8bf66a380b8640a798f24000000000000000000000000946da73853fac1441998fd5c14c7029988ffe9b80000000000000000000000000404ebf40d76a32b0dccdf13f3ff8a9a7e0fad1900000000000000000000000000000000000000000000000000000000000001f42da0178b57fae97031c6ba540b293f3847589d1890ab0b62b56b4ca4a87786800e7ba03d9e1fcf9f21fa759b832c588bee92d5b99fbff5ada6a72c71f5336c9a22c961\"],\"id\":605,\"jsonrpc\":\"2.0\"}", requestMethod="POST", url="http://geth.goerli.ethnodes.brainbot.com:8545", code=SERVER_ERROR, version=web/5.0.12)
        at Logger.makeError (/lc/index.js:7523:21)
        at Logger.throwError (/lc/index.js:7532:20)
        at /lc/index.js:19174:32
        at step (/lc/index.js:18942:23)
        at Object.next (/lc/index.js:18923:53)
        at fulfilled (/lc/index.js:18914:58)
        at runMicrotasks (<anonymous>)
        at processTicksAndRejections (internal/process/task_queues.js:93:5) {
      reason: 'processing response error',
      code: 'SERVER_ERROR',
      body: '{"jsonrpc":"2.0","id":605,"error":{"code":-32000,"message":"replacement transaction underpriced"}}\n',
      error: Error: replacement transaction underpriced
          at getResult (/lc/index.js:12174:21)
          at processJsonFunc (/lc/index.js:19217:22)
          at /lc/index.js:19149:46
          at step (/lc/index.js:18942:23)
          at Object.next (/lc/index.js:18923:53)
          at fulfilled (/lc/index.js:18914:58)
          at runMicrotasks (<anonymous>)
          at processTicksAndRejections (internal/process/task_queues.js:93:5) {
        code: -32000,
        data: undefined
      },
      requestBody: '{"method":"eth_sendRawTransaction","params":["0xf8ca8085012a05f20083019092943ea2a1fed7fdef300da19e97092ce8fdf8bf66a380b8640a798f24000000000000000000000000946da73853fac1441998fd5c14c7029988ffe9b80000000000000000000000000404ebf40d76a32b0dccdf13f3ff8a9a7e0fad1900000000000000000000000000000000000000000000000000000000000001f42da0178b57fae97031c6ba540b293f3847589d1890ab0b62b56b4ca4a87786800e7ba03d9e1fcf9f21fa759b832c588bee92d5b99fbff5ada6a72c71f5336c9a22c961"],"id":605,"jsonrpc":"2.0"}',
      requestMethod: 'POST',
      url: 'http://geth.goerli.ethnodes.brainbot.com:8545'
    },
    method: 'sendTransaction',
    transaction: {
      nonce: 0,
      gasPrice: BigNumber { _hex: '0x012a05f200', _isBigNumber: true },
      gasLimit: BigNumber { _hex: '0x019092', _isBigNumber: true },
      to: '0x3EA2a1fED7FdEf300DA19E97092Ce8FdF8bf66A3',
      value: BigNumber { _hex: '0x00', _isBigNumber: true },
      data: '0x0a798f24000000000000000000000000946da73853fac1441998fd5c14c7029988ffe9b80000000000000000000000000404ebf40d76a32b0dccdf13f3ff8a9a7e0fad1900000000000000000000000000000000000000000000000000000000000001f4',
      chainId: 5,
      v: 45,
      r: '0x178b57fae97031c6ba540b293f3847589d1890ab0b62b56b4ca4a87786800e7b',
      s: '0x3d9e1fcf9f21fa759b832c588bee92d5b99fbff5ada6a72c71f5336c9a22c961',
      from: '0x946da73853FAC1441998Fd5c14c7029988FFE9B8',
      hash: '0x6e280ec913da91cecb9377a5bd3f6a80bd3b4a87e09b77856e1a14a45dd5a928'
    },
    transactionHash: '0x6e280ec913da91cecb9377a5bd3f6a80bd3b4a87e09b77856e1a14a45dd5a928'
  }
}
```

[scenario-player-run_pfs1_get_a_simple_path_2021-01-22T00:59:34.log](https://github.com/raiden-network/light-client/files/5855585/scenario-player-run_pfs1_get_a_simple_path_2021-01-22T00.59.34.log)

[pfs1_node1_node3_22jan2021.tar.gz](https://github.com/raiden-network/light-client/files/5855588/pfs1_node1_node3_22jan2021.tar.gz)

**Comment by agatsoh** [comment]
Occured again in PFS9 today

**Comment by agatsoh** [comment]
Occurred in BF6 23rd Jan 2021  

BF6 gets transfer expired error node reveal this is the similar kind of error that we get here 
**Transaction gas price supplied is too low. There is another transaction with same nonce in the queue. Try increasing the gas price or incrementing the nonce**
```
2021-01-24T04:35:15.207Z [info] retryWhile: retrying {
  count: 0,
  interval: 5000,
  error: Error: processing response error (body="{\"jsonrpc\":\"2.0\",\"error\":{\"code\":-32010,\"message\":\"Transaction gas price supplied is too low. There is another transaction with same nonce in the queue. Try increasing the gas price or incrementing the nonce.\"},\"id\":590}\n", error={"code":-32010}, requestBody="{\"method\":\"eth_sendRawTransaction\",\"params\":[\"0xf8c98084773594008301909c943ea2a1fed7fdef300da19e97092ce8fdf8bf66a380b8640a798f24000000000000000000000000757eca3c6f2e5f7e0d3274f906a4a6e8d8b85b77000000000000000000000000810e0726f3372f073af461789f5d085f937897d500000000000000000000000000000000000000000000000000000000000001f42ea00fa3f58fa0117d9102a5757fd7bfe1daa6f2d3b1d9d7d9f69a7acc7ee664bbdda0221161eb42bf476d85229d02d3df1655262bf0f68ef8870355b915bdf8c57436\"],\"id\":590,\"jsonrpc\":\"2.0\"}", requestMethod="POST", url="http://parity.goerli.ethnodes.brainbot.com:8545", code=SERVER_ERROR, version=web/5.0.12)
      at Logger.makeError (/lc/index.js:7523:21)
      at Logger.throwError (/lc/index.js:7532:20)
      at /lc/index.js:19174:32
      at step (/lc/index.js:18942:23)
      at Object.next (/lc/index.js:18923:53)
      at fulfilled (/lc/index.js:18914:58)
      at processTicksAndRejections (internal/process/task_queues.js:93:5) {
    reason: 'processing response error',
    code: 'SERVER_ERROR',
    body: '{"jsonrpc":"2.0","error":{"code":-32010,"message":"Transaction gas price supplied is too low. There is another transaction with same nonce in the queue. Try increasing the gas price or incrementing the nonce."},"id":590}\n',
    error: Error: Transaction gas price supplied is too low. There is another transaction with same nonce in the queue. Try increasing the gas price or incrementing the nonce.
        at getResult (/lc/index.js:12174:21)
        at processJsonFunc (/lc/index.js:19217:22)
        at /lc/index.js:19149:46
        at step (/lc/index.js:18942:23)
        at Object.next (/lc/index.js:18923:53)
        at fulfilled (/lc/index.js:18914:58)
        at processTicksAndRejections (internal/process/task_queues.js:93:5) {
      code: -32010,
      data: undefined
    },
    requestBody: '{"method":"eth_sendRawTransaction","params":["0xf8c98084773594008301909c943ea2a1fed7fdef300da19e97092ce8fdf8bf66a380b8640a798f24000000000000000000000000757eca3c6f2e5f7e0d3274f906a4a6e8d8b85b77000000000000000000000000810e0726f3372f073af461789f5d085f937897d500000000000000000000000000000000000000000000000000000000000001f42ea00fa3f58fa0117d9102a5757fd7bfe1daa6f2d3b1d9d7d9f69a7acc7ee664bbdda0221161eb42bf476d85229d02d3df1655262bf0f68ef8870355b915bdf8c57436"],"id":590,"jsonrpc":"2.0"}',
    requestMethod: 'POST',
    url: 'http://parity.goerli.ethnodes.brainbot.com:8545',
    transaction: {
      nonce: 0,
      gasPrice: BigNumber { _hex: '0x77359400', _isBigNumber: true },
      gasLimit: BigNumber { _hex: '0x01909c', _isBigNumber: true },
      to: '0x3EA2a1fED7FdEf300DA19E97092Ce8FdF8bf66A3',
      value: BigNumber { _hex: '0x00', _isBigNumber: true },
      data: '0x0a798f24000000000000000000000000757eca3c6f2e5f7e0d3274f906a4a6e8d8b85b77000000000000000000000000810e0726f3372f073af461789f5d085f937897d500000000000000000000000000000000000000000000000000000000000001f4',
      chainId: 5,
      v: 46,
      r: '0x0fa3f58fa0117d9102a5757fd7bfe1daa6f2d3b1d9d7d9f69a7acc7ee664bbdd',
      s: '0x221161eb42bf476d85229d02d3df1655262bf0f68ef8870355b915bdf8c57436',
      from: '0x757ECa3c6f2E5f7E0D3274F906a4A6e8d8b85b77',
      hash: '0x0cde3bd1a582c1c906d40241c2643a84fa0d87fe91393edbff05496b8616b578'
    },
    transactionHash: '0x0cde3bd1a582c1c906d40241c2643a84fa0d87fe91393edbff05496b8616b578'
  }
}
```

[scenario-player-run_bf6_stress_hub_node_2021-01-24T04:33:09.log](https://github.com/raiden-network/light-client/files/5865832/scenario-player-run_bf6_stress_hub_node_2021-01-24T04.33.09.log)


[node_4617_003.tar.gz](https://github.com/raiden-network/light-client/files/5865843/node_4617_003.tar.gz)

**Comment by andrevmatos** [comment]
Fixed by 1f37f9f
